apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -34,13 +36,32 @@ const ProductCodec = t.strict({
   }),
 });
 
+const NewsItemCodec = t.strict({
+  attributes: t.strict({
+    comingSoon: t.boolean,
+    title: t.string,
+    link: LinkCodec,
+    publishedAt: t.string,
```

Comment:
For this suggestion you have to import `io-ts-types` as following `import * as tt from 'io-ts-types';`

```suggestion
    publishedAt: tt.DateFromISOString,
```

Reply:
Nice